filament/backend/src/opengl/OpenGLContext.h

```diff
@@ -155,6 +155,7 @@ class OpenGLContext {
         bool WEBGL_compressed_texture_etc = false;
         bool WEBGL_compressed_texture_s3tc = false;
         bool WEBGL_compressed_texture_s3tc_srgb = false;
+        bool EXT_external_buffer = false;
```

Comment:
What's this, a leftover from https://github.com/shapr3d/filament/pull/1? External buffers aren't that relevant at this point, the ticket about those is [GFX-594](https://shapr3d.atlassian.net/browse/GFX-594).